blueprints/codes/latex_formula.py

```diff
@@ -0,0 +1,57 @@
+"""Latex formula representation."""
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class LatexFormula:
+    """Latex formula representation.
+    Depending on the context this could include the unit, the formula, the result, etc.
+
+    Attributes
+    ----------
+    return_symbol: str
+        The symbol to return
+    result: str
+        The result of the formula
+    equation: str, default ""
+        The formula with symbols
+    numeric_equation: str, default ""
+        The formula with values (numbers)
+    comparison_operator_label: str, default "="
+        The label for the comparison operators between the return symbol and the result.
+        Could be changed for inequalities.
+    """
+
+    return_symbol: str
+    result: str
```

Comment:
I think this should be a float instead. Transforming to a str is the responsibility of the class itself.

Reply:
as it only contains strings and in theorie should be possible for any formula (including string results) i suggest we stick to str for all subresults.